valor.css

```diff
@@ -1,10 +1,16 @@
+html {
+	background: #fff;
+	color: #000;
+}
 
 namaka-ban::part(container),
 namaka-unban::part(container),
 namaka-edit-comment::part(container),
 namaka-author-reply::part(container),	
 namaka-archive-comment::part(container),
 namaka-board-stats::part(container),
+namaka-comments::part(container),
+namaka-board::part(container),
 namaka-write-a-comment::part(container) {	
```

Comment:
i don't think we should use this `::part(container)` strategy

if it's possible, i think we should actually try to apply these styles onto the host elements directly, ie

```css
namaka-ban,
namaka-unban
/*...*/ {
  background: #FFAF02;
}
```

let's at least check if this simpler approach is possible

Reply:
even though i think we should avoid the `::part(container)` pattern,

it's an example of where we could use `:is` to reduce a little bit of repetition :monocle_face: 

```css
:is(
    namaka-ban,
    namaka-unban,
    namaka-edit-comment,
    namaka-author-reply,
    namaka-archive-comment,
    namaka-board-stats,
    namaka-comments,
    namaka-board,
    namaka-write-a-comment
  )::part(container) {
  background: #FFAF02;
}
```

Reply:
The second method works and it's been fixed